File: Trabajo4/RecorridoFilas.py
FILAS = 3
COLUMNAS = 4

matriz = [
    [1, 2, 3, 4],
    [5, 6, 7, 8],
    [9, 10, 11, 12]
]

# Recorrer por renglones (filas)
print("Recorrido por renglones:")
for fila in range(FILAS):
    print(f"Renglón {fila}: ", end="")
    for col in range(COLUMNAS):
        print(f"{matriz[fila][col]} ", end="")
    print()

File: Trabajo4/RecorridoFilas.java
public class Main {
    static final int FILAS = 3;
    static final int COLUMNAS = 4;
    
    public static void main(String[] args) {
        int[][] matriz = {
            {1, 2, 3, 4},
            {5, 6, 7, 8},
            {9, 10, 11, 12}
        };
        
        // Recorrer por renglones (filas)
        System.out.println("Recorrido por renglones:");
        for (int fila = 0; fila < FILAS; fila++) {
            System.out.print("Renglón " + fila + ": ");
            for (int col = 0; col < COLUMNAS; col++) {
                System.out.print(matriz[fila][col] + " ");
            }
            System.out.println();
        }
    }
}